artemis/modules/wordpress_plugins.py

```diff
@@ -62,6 +64,28 @@
 ]
 
 
+def _is_version_larger(v1: str, v2: str) -> bool:
```

Comment:
what if version has character or string in it, like `v6.1.0a`

Reply:
we compare numbers and fall back to lexicographical compare if the values aren't numeric